doc/OnlineDocs/related_packages.rst

```diff
@@ -0,0 +1,43 @@
+|  Galini                  | https://github.com/cog-imperial/galini                  |  An extensible, Python-based MIQCQP Solver  |
++--------------------------+---------------------------------------------------------+---------------------------------------------+
+|  IDAES                   | https://github.com/idaes/idaes-pse                      |  Institute for the Design of Advanced       |
+|                          |                                                         |  Energy Systems                             |
++--------------------------+---------------------------------------------------------+---------------------------------------------+
+|  mpi-sppy                | https://github.com/pyomo/mpi-sppy                       |  Parallel solution of                       |
+|                          |                                                         |  stochastic programming problems            |
++--------------------------+---------------------------------------------------------+---------------------------------------------+
+|  PAO                     | https://github.com/or-fusion/pao                        |  Formulation and solution of multilevel     |
+|                          |                                                         |  optimization problems                      |
++--------------------------+---------------------------------------------------------+---------------------------------------------+
+|  Parapint                | https://github.com/parapint/parapint                    |  Parallel solution of structured            |
+|                          |                                                         |  NLPs.                                      |
++--------------------------+---------------------------------------------------------+---------------------------------------------+
+|  Prescient               | https://github.com/grid-parity-exchange/prescient       |  Parallel solution of structured            |
+|                          |                                                         |  NLPs.                                      |
++--------------------------+---------------------------------------------------------+---------------------------------------------+
+|  PyPSA                   | https://github.com/pypsa/pypsa                          |  Python for Power system Analysis           |
++--------------------------+---------------------------------------------------------+---------------------------------------------+
+|  Suspect                 | https://github.com/cog-imperial/suspect                 |  FBBT and convexity detection               |
++--------------------------+---------------------------------------------------------+---------------------------------------------+
+
+
+.. rubric:: Footnotes
+
+.. [#f1] Please note that Pyomo does not vet the above packages.
```

Comment:
What about "Please note that the Pyomo team does not evaluate or endorse the packages listed above"?  We technically vet the list, as they were all either added by us, or (int the future) we approved the request to add them through a PR.

Reply:
Another great suggestion.